src/commonTest/kotlin/fr/acinq/lightning/channel/TestsHelper.kt

```diff
@@ -142,64 +217,64 @@ object TestsHelper {
         alicePushAmount: MilliSatoshi = TestConstants.alicePushAmount,
         bobPushAmount: MilliSatoshi = TestConstants.bobPushAmount,
         zeroConf: Boolean = false,
-    ): Triple<Normal, Normal, Transaction> {
+    ): Triple<LNChannel<Normal>, LNChannel<Normal>, Transaction> {
         val (alice, channelReadyAlice, bob, channelReadyBob) = WaitForChannelReadyTestsCommon.init(channelType, aliceFeatures, bobFeatures, currentHeight, aliceFundingAmount, bobFundingAmount, alicePushAmount, bobPushAmount, zeroConf)
         val (alice1, actionsAlice1) = alice.process(ChannelCommand.MessageReceived(channelReadyBob))
-        assertIs<Normal>(alice1)
+        assertIs<LNChannel<Normal>>(alice1)
```

Comment:
We could avoid repeating `LNChannel` everywhere by offering a helper method that does the same as `AssertIs` but for the channel state and can be directly called on an `LNChannel` (a slightly annoying find/replace once, but future test writing will be better).

Reply:
Great idea, and would have saved me time on the first iteration. Will give it a try.

Reply:
Hmm, for some reason this doesn't work:
```kotlin
inline fun <reified S : ChannelState> assertIsState(value: Any) = assertIs<LNChannel<S>>(value)
```
It doesn't do an implicit cast like the original `assertIs`.

Reply:
> But instead of doing fancy stuff we could also simply go with:
> assertIs<Normal>(alice1.state)

If we do that we won't benefit from the Kotlin's smart-cast, will we? What's nice when doing `assertIs<Normal>(state)` is that `state` in smart-casted to `Normal` for the rest of the scope, but I don't think it works with nested arguments...

If this is too cumbersome that's something we can fix in a follow-up PR, it's not ideal but I don't want this to block the serialization update.

Reply:
> If we do that we won't benefit from the Kotlin's smart-cast, will we? What's nice when doing assertIs<Normal>(state) is that state in smart-casted to Normal for the rest of the scope, but I don't think it works with nested arguments...

It does, actually. But the way we then use the object make us lose this cast unfortunately, so the change isn't trivial. Let's leave it like this for now.